packages/taquito-sapling/src/sapling-tx-builder/sapling-module-wrapper.ts

```diff
@@ -0,0 +1,146 @@
+import * as sapling from '@airgap/sapling-wasm';
```

Comment:
@zainen it's the sapling-wasm library that help us do some sapling-specific stuff. It's a wrapper of a Rust implementation of Sapling based methods

Reply:
sorry wasnt clear I meant the whole file for sapling-module-wrapper.ts not just the one file just picked the first line

Reply:
It created a separate class to wrap the "sapling" instance just to be able to write my unit tests. Some of the methods from the sapling-wasm library require a sapling context which is a number that is always different, and the returned values are different from one run to another because of this context. I could not properly test without wrapping and mocking the class.